gulpfile.js

```diff
@@ -30,9 +29,39 @@ var COPY_DIRS = [
   'vendor/bootstrap/css/**',
   'vendor/bootstrap/fonts/**',
 ];
+var COMPILED_DIR = './dist/';
 
 var LESS_FILES = './less/**/*.less';
 
+function webpackTask(options) {
+  console.log(options)
+  options = options || {};
+  var srcFile = './lib/index-static.jsx';
+  var outputPath = path.join(__dirname, COMPILED_DIR, 'js');
+  fs.removeSync(outputPath);
+  var outputName = 'bundle.js';
+  var config = require('./webpack.config');
```

Comment:
hmm, since we are essentially dynamically creating our webpack config, perhaps we should just do away with `webpack.config.js` entirely?